controllers/remove_services.go

```diff
@@ -48,6 +48,7 @@ func (u removeServices) reconcile(ctx context.Context, r *FoundationDBClusterRec
 		return &requeue{curError: err}
 	}
 
+	log.Info("Deleting service", "name", existingService.Name)
```

Comment:
That should be a debug statement not an info statement. I would prefer if we also log the corresponding namespace and cluster name.